flags/14.0/vttablet.txt

```diff
@@ -195,32 +101,34 @@ Usage of vttablet:
       --enable_hot_row_protection                                        If true, incoming transactions for the same row (range) will be queued and cannot consume all txpool slots.
       --enable_hot_row_protection_dry_run                                If true, hot row protection is not enforced but logs if transactions would have been queued.
       --enable_lag_throttler                                             If true, vttablet will run a throttler service, and will implicitly enable heartbeats
-      --enable_query_plan_field_caching                                  This option fetches & caches fields (columns) when storing query plans (default true)
+      --enable_query_plan_field_caching                                  (DEPRECATED) This option fetches & caches fields (columns) when storing query plans (default true)
```

Comment:
we should `MarkDeprecated` for this flag.

Reply:
fixed